x-pack/plugins/security/server/lib/authorization/mode.test.js

```diff
@@ -6,30 +6,76 @@
 
 import { authorizationModeFactory } from './mode';
 
-const createMockXpackInfoFeature = (allowRbac) => {
-  return {
-    getLicenseCheckResults() {
-      return {
-        allowRbac
-      };
-    }
-  };
-};
-
-describe(`#useRbac`, () => {
+class MockXPackInfoFeature {
```

Comment:
__note:__ there is no need to change anything as current approach looks good to me, just wanted to share another option where `mockSetAllowRbac` can be replaced with `Jest` built-in mock functionality:
```ts
const createMockXpackInfoFeature = (allowRbac = false) => {
  return { getLicenseCheckResults: jest.fn().mockReturnValue({ allowRbac }) };
};
.....
mockXpackInfoFeature.getLicenseCheckResults.mockReturnValue({ allowRbac: true });
.....

Reply:
I think this is a lot better, I will make this change.